src/address.js

```diff
@@ -1,7 +1,8 @@
 const artblocks = {
   "mainnet": {
    "v0": "0x059edd72cd353df5106d2b9cc5ab83a52287ac3a",
-   "v1": "0xa7d8d9ef8d8ce8992df33d8b8cf4aebabd5bd270" 
+   "v1": "0xa7d8d9ef8d8ce8992df33d8b8cf4aebabd5bd270",
```

Comment:
Nit: can we add a comment specifying that "v2" is not present as the "v2" spec was only applicable for Engine?